src/avrintel.h

```diff
@@ -9,7 +9,7 @@
  * meta-author Stefan Rueger <stefan.rueger@urclocks.com>
  *
  * v 1.1
- * 30.08.2022
+ * 04.11.2022
```

Comment:
Just wondering if this is good to change to 04-Nov-2022 as it is ambiguous and may get interpreted wrongly by US based people